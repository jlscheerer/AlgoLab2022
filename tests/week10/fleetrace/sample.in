5
2 3 6
0 0 6
0 1 3
0 2 4
1 0 9
1 1 2
1 2 8
4 3 7
0 0 3
0 1 3
1 0 3
1 1 3
2 0 3
2 1 3
2 2 3
4 5 8
0 0 5
0 1 5
1 0 5
2 0 5
2 1 5
3 1 5
3 2 5
3 3 5
4 3 7
0 0 4
0 1 5
1 0 1
1 1 2
2 0 6
2 1 10
2 2 7
4 5 8
0 0 4
0 1 5
1 0 4
2 0 6
2 1 4
3 1 10
3 2 5
3 3 6
