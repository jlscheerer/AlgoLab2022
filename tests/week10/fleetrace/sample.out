14
9
15
14
17
